PlaywrightSolar/Tests/ProductDetailsTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using PlaywrightSolar.PageObjects;
+
+namespace PlaywrightSolar.Tests;
+
+internal class ProductDetailsTest : UiTestFixture
+{
+    private InvertersPage _invertersPage;
+    
+    [SetUp]
+    public void SetupInvertersPage()
+    {
+        _invertersPage = new InvertersPage(Page);
+    }
+    
+    [Test]
+    public async Task CompareProductNames()
+    {
+        // Arrange
+        await _invertersPage.GoToInvertersPage();
+        
+        // Act
+        var productNameFromCard = await _invertersPage.GetProductName(5);
```

Comment:
5 is magic number again. In fact, when I read this test, I can't be sure if this 5 and next 5 are the same and if I need to change this test, I need to investigate implementation of both GetProductName() and GoToProductDetails() to ensure that these are the same 5
it should be
var productIndex = 5;
GetProductName(productIndex);
GoToProductDetails(productIndex);